rustls/src/msgs/deframer.rs

```diff
@@ -455,6 +489,24 @@ impl DeframerBuffer for DeframerVecBuffer {
     }
 }
 
+impl DeframerBuffer for DeframerSliceBuffer<'_> {
```

Comment:
This impl should live below the `DeframerSliceBuffer`'s inherent `impl`, and above the `DeframerBuffer` defintion.